### PR TITLE
BUG: UPDATE HIERACHY OF SHIFT PERMISSION APPROVER

### PR DESCRIPTION
**Pull request by NoBoneZ**
## Is this a Feature, Chore or Bug?
- [] Feature
- [] Chore
- [X] Bug

## Clearly and concisely describe the feature, chore or bug.
To update the hierachy of the shift permission approver, such that, if there is no reporst to(less likely), it picks the operations site supervisor

## Analysis and design (optional)
Analyse and attach the design documentation

## Solution description
Describe your code changes in detail for reviewers.
Added a code block that handles the hierachy 

## Is there a business logic within a doctype?
    - [] Yes
    - [] No

## Output screenshots (optional)
Post the output screenshots, if a UI is affected or added due to this feature.

## Areas affected and ensured
List out the areas affected by your code changes.
Shift permission

## Is there any existing behavior change of other features due to this code change?
Mention Yes or No. If Yes, provide the appropriate explanation.

## Did you test with the following dataset?
- [] Existing Data
- [x] New Data

## Was child table created?
    - [] is attachment required?
        did you test attachment
## Did you delete custom field?
    - [] Yes
    - [x] No
        If yes, did you write a delete patch?

## Is patch required?
- [] Yes
- [x] No
    ## Was the patch test?

## Which browser(s) did you use for testing?
  - [x] Chrome
  - [] Safari
  - [] Firefox

### CODE REVIEW

**Review by pjamsheer — APPROVED** [review]

**Comment by pjamsheer** [comment]
@AbrahamAdekunle fix the conflict

**Comment by NoBoneZ** [comment]
> @AbrahamAdekunle fix the conflict

@pjamsheer , resolved

File: one_fm/one_fm/doctype/hr_and_payroll_additional_settings/hr_and_payroll_additional_settings.json [hunk 1]
```diff
@@ -279,26 +277,12 @@
    "fieldname": "section_break_nkbfu",
    "fieldtype": "Section Break",
    "hide_border": 1
-  },
```
Comment (pjamsheer) [inline one_fm/one_fm/doctype/hr_and_payroll_additional_settings/hr_and_payroll_additional_settings.json]: @AbrahamAdekunle why you removed this fields

**Review by pjamsheer — CHANGES_REQUESTED** [review]

File: one_fm/one_fm/doctype/hr_and_payroll_additional_settings/hr_and_payroll_additional_settings.json (hunk 1 above)
Comment (NoBoneZ) [reply one_fm/one_fm/doctype/hr_and_payroll_additional_settings/hr_and_payroll_additional_settings.json]: @pjamsheer , resolved

**Review by pjamsheer — APPROVED** [review]

**Pull request merged**